internal/snmp/translate.go

```diff
@@ -127,6 +134,20 @@ func TrapLookup(oid string) (e MibEntry, err error) {
 	}
 	e.MibName = e.OidText[:i]
 	e.OidText = e.OidText[i+2:]
+
+	var s int
+	if len(foundOid) < len(givenOid) {
+		for i := range foundOid {
+			if foundOid[i] == givenOid[i] {
+				s = s + 1
+			}
+		}
+	}
+
+	givenOid = givenOid[s:]
+
+	e.OidText = e.OidText + "." + strings.Join(givenOid, ".")
```

Comment:
```suggestion
	e.OidText = e.OidText + "." + givenOid
```

Also wouldn't this resulting in `e.OidText` to always end with `.`?